django_log_formatter_asim/__init__.py

```diff
@@ -1,176 +1,126 @@
-                    zipkin_header
-                ]  # noqa E501
-
-        logger_event = self._get_event_base(
-            extra_labels=extra_labels,
-        )
-
-        parsed_url = urlparse(self.record.request.build_absolute_uri())
-
-        ip = self._get_ip_address(self.record.request)
-
-        request_bytes = len(self.record.request.body)
-
-        logger_event.url(
-            path=parsed_url.path,
-            domain=parsed_url.hostname,
-        ).source(
-            ip=self._get_ip_address(self.record.request)
-        ).http_response(status_code=getattr(self.record, "status_code", None)).client(
-            address=ip,
-            bytes=request_bytes,
-            domain=parsed_url.hostname,
-            ip=ip,
-            port=parsed_url.port,
-        ).http_request(
-            body_bytes=request_bytes,
-            body_content=self.record.request.body,
-            method=self.record.request.method,
-        )
-
-        user_agent_string = getattr(
-            self.record.request.headers,
-            "user_agent",
-            None,
-        )
-
-        if not user_agent_string and "HTTP_USER_AGENT" in self.record.request.META:  # noqa E501
-            user_agent_string = self.record.request.META["HTTP_USER_AGENT"]
-
-        # Check for use of django-user_agents
-        if getattr(self.record.request, "user_agent", None):
-            logger_event.user_agent(
-                device={
-                    "name": self.record.request.user_agent.device.family,
-                },
-                name=self.record.request.user_agent.browser.family,
-                original=user_agent_string,
-                version=self.record.request.user_agent.browser.version_string,
-            )
-        elif user_agent_string:
-            logger_event.user_agent(
-                original=user_agent_string,
-            )
-
-        if getattr(self.record.request, "user", None):
-            if getattr(settings, "DLFE_LOG_SENSITIVE_USER_DATA", False):
-                # Defensively check for full name due to possibility of custom user app
-                try:
-                    full_name = self.record.request.user.get_full_name()
-                except AttributeError:
-                    full_name = None
-
-                # Check user attrs to account for custom user apps
-                logger_event.user(
-                    email=getattr(self.record.request.user, "email", None),
-                    full_name=full_name,
-                    name=getattr(self.record.request.user, "username", None),
-                    id=getattr(self.record.request.user, "id", None),
-                )
-            else:
-                logger_event.user(
-                    id=getattr(self.record.request.user, "id", None),
-                )
+    def __init__(self, record):
+        super().__init__(record)
 
-        return logger_event
+    def get_log_dict(self):
+        log_dict = super().get_log_dict_base()
```

Comment:
As @antroy-madetech suggested, it makes senes to use super() and ditch the `_base()` method. 

```
def get_log_dict(self):
      log_dict = super().get_log_dict()
```

Reply:
I only half did the change before. Just completed it. We don't even need to declare the method in this class.